libs/wire-api/src/Wire/API/Conversation/Member.hs

```diff
@@ -169,26 +170,31 @@ newtype MutedStatus = MutedStatus {fromMutedStatus :: Int32}
   deriving (FromJSON, ToJSON, S.ToSchema) via Schema MutedStatus
 
 data OtherMember = OtherMember
-  { omId :: UserId,
+  { omQualifiedId :: Qualified UserId,
     omService :: Maybe ServiceRef,
     omConvRoleName :: RoleName
   }
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform OtherMember)
+  deriving (FromJSON, ToJSON, S.ToSchema) via Schema OtherMember
+
+omId :: OtherMember -> UserId
+omId = qUnqualified . omQualifiedId
```

Comment:
This function could be a cause of logical errors. If we really want a utility like this, we should call it `omLocalId` or something. If we just want to save on integration tests tripping over lack of this function, perhaps we could write this function in integration tests?

Reply:
I removed the function entirely, as it's only used in the schema instance below (I replaced all other occurrences)